ci/test/00_setup_env.sh

```diff
@@ -34,6 +34,7 @@ export USE_BUSY_BOX=${USE_BUSY_BOX:-false}
 export RUN_UNIT_TESTS=${RUN_UNIT_TESTS:-true}
 export RUN_FUNCTIONAL_TESTS=${RUN_FUNCTIONAL_TESTS:-true}
 export RUN_FUZZ_TESTS=${RUN_FUZZ_TESTS:-false}
+export DOCKER_NAME=${DOCKER_NAME:-ubuntu_18_04}
```

Comment:
If you feel extra fancy, you might even rename the variable to `CONTAINER_NAME` to not cause confusion with `DOCKER_NAME_TAG`.